exonum/build.rs

```diff
@@ -15,6 +19,19 @@ fn main() {
     let mut file = File::create(dest_path).expect("Unable to create output file");
     file.write_all(user_agent.as_bytes())
         .expect("Unable to data to file");
+
+    protoc_rust::run(protoc_rust::Args {
+        out_dir: "src/encoding/protobuf",
+        input: &[
```

Comment:
@dvush What do you think about to create a function for scanning path `src/encoding/protobuf/proto/` for files with `.proto` extension. It would be useful in case if you add a new proto file then this section could be unchanged.

Reply:
My two cents: I'm personally not a fan of this approach, for reasons similar to why Rust code style does not favor glob imports (`use module::*` to bring in all the stuff from a module). Explicit source file list makes it easier to track what's actually compiled and enables grepping the source code for usages.